macro/src/lib.rs

```diff
@@ -239,14 +261,9 @@ fn fetch_metadata(args: &RuntimeMetadataArgs) -> Result<subxt_codegen::Metadata,
                 .and_then(|b| subxt_codegen::Metadata::decode(&mut &*b).map_err(Into::into))
                 .map_err(|e| e.into_compile_error())?
         }
-        (None, None) => {
-            abort_call_site!(
-                "One of 'runtime_metadata_path' or 'runtime_metadata_insecure_url' must be provided"
-            )
-        }
-        (Some(_), Some(_)) => {
+        _ => {
```

Comment:
This warning message isn't accurate if `--feature runtime-path` is enabled, so it could be improved by "feature-gating" this such as

```rust
match path {
    #[cfg(feature = "runtime-path")
    _ => {
           abort_call_site!(
                "Only one of 'runtime_metadata_path', 'runtime_metadata_insecure_url'  or  'runtime_path` can be provided"
                "Exclusively one of 'runtime_metadata_path' or 'runtime_metadata_insecure_url' must be provided"
            )
    }
    #[cfg(not(feature = "runtime-path"))
    _ => {
            abort_call_site!(
                "Only one of 'runtime_metadata_path' or 'runtime_metadata_insecure_url' can be provided"
                "Exclusively one of 'runtime_metadata_path' or 'runtime_metadata_insecure_url' must be provided"
            )
    }
};
```

Reply:
Yeah, good catch! changed per suggestion.